website/content/docs/plugins.mdx

```diff
@@ -16,13 +16,13 @@ how to write plugins, see the [Extending Waypoint guide](/waypoint/docs/extendin
 Waypoint ships with dozens of built-in plugins. These plugins are available
 out-of-the-box with Waypoint and are built directly on the same plugin system
 as external plugins. To see a list of supported plugins, see the
-[plugin documentation](/waypoint/plugins).
+[integrations library](/waypoint/integrations).
 
 ## Available Plugins
 
-You can see a list of [all built-in plugins here](/waypoint/plugins).
+You can see a list of all built-in plugins in our [integrations library](/waypoint/integrations).
 
-We do not maintain a list of external plugins currently.
+We do not maintain a list of external plugins currently, but plan to incorporate them into our integrations library in the future.
```

Comment:
I would say this entire PR is mostly non-controversial, with a potential exception of these changes.

I call out our future plan to incorporate external plugins into our integrations library.  This is more of the same of a discussion that we had regarding our communications of this release, but I want to double tap here that I'm doubling down on the decision that we made to include this language in our comms by including it in the docs.

Reply:
I would say that this is our shim for external plugins - `[all built-in plugins here](/waypoint/plugins)`
So maybe add that to the second sentence? Something like, "Currently the list of external plugins is here, and we plan to incorporate them .. etc"

Reply:
Can pull in the documentation here https://developer.hashicorp.com/waypoint/plugins#community-plugin-list to talk about the temporary external plugins list, then update that last sentence to `We maintain a list of external plugins [here], and plan to incorporate them...` etc.

Reply:
That's a good idea!  I'll do that

Reply:
I see I missed updating this page you are updating here, when I updated https://developer.hashicorp.com/waypoint/plugins, so yay for consolidating these infos.